aries-site/src/examples/components/tag/TagAnatomy.js

```diff
@@ -0,0 +1,130 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Grid, Diagram, Stack, Text, ThemeContext } from 'grommet';
+import { FormClose } from 'grommet-icons';
+import { Annotation } from '../namevaluelist/NameValueListAnatomy';
+
+const color = 'border';
+const anchor = 'vertical';
+const thickness = 'hair';
+const type = 'direct';
+
+const connections = [
+  {
+    anchor,
+    type,
+    color,
+    thickness,
+    fromTarget: 'border-annotation',
+    toTarget: 'border',
+  },
+  {
+    anchor: 'horizontal',
+    type,
+    color,
+    thickness,
+    fromTarget: 'name-annotation',
+    toTarget: 'tagName-2',
+  },
+  {
+    anchor,
+    type,
+    color,
+    thickness,
+    fromTarget: 'value-annotation',
+    toTarget: 'value',
+  },
+  {
+    anchor: 'horizontal',
+    type,
+    color,
+    thickness,
+    fromTarget: 'close-annotation',
+    toTarget: 'close',
+  },
+];
+
+export const AnatomyGrid = ({ ...rest }) => (
+  <Grid
+    columns={['xsmall', 'small', 'xsmall']}
+    gap={{ column: 'small', row: 'medium' }}
+    justify="center"
+    {...rest}
+  />
+);
+
+export const AnatomyBox = ({ id, nameId, valueId, closeId }) => {
+  const theme = useContext(ThemeContext);
+
+  return (
+    <Box
+      direction="row"
+      gap="xxsmall"
+      id={id}
+      border={theme.tag.border}
+      pad={theme.tag.pad}
+      round={theme.tag.round}
+    >
+      <Box id={nameId}>Location</Box>
+      {valueId && closeId && (
+        <>
+          <Box>:</Box>
+          <Box id={valueId}>
+            <Text {...theme.tag.value}>NY_USA</Text>
+          </Box>
+          <Box id={closeId}>
+            <FormClose />
+          </Box>
```

Comment:
Any reason ids can't be placed on the Text and FormClose? The Boxes seem extraneous to me.

```suggestion
          <Box id={valueId}>
            <Text {...theme.tag.value}>NY_USA</Text>
          </Box>
          <Box id={closeId}>
            <FormClose />
          </Box>
```